src/datasources/entity-ds-react/EntityHelper.tsx

```diff
@@ -153,6 +153,20 @@ export const queryEntity = async (label: string | undefined, filter: API.Filter,
     return queryResult
 }
 
+export const queryProperties = async (queryWithNodeFilterWrapper: string, client: ClientDelegate) => {
```

Comment:
I'm getting  the following error under this scenario. Probably this can be fixed by @synqotik  before merging to the features branch.  Otherwise look good.

<img width="519" alt="Screen Shot 2022-11-21 at 10 48 38 AM" src="https://user-images.githubusercontent.com/3297292/203098493-1b4ac06e-ec57-4450-bd0f-52177745a634.png">
<img width="1106" alt="Screen Shot 2022-11-21 at 10 47 18 AM" src="https://user-images.githubusercontent.com/3297292/203098837-5edfdcc9-3cc6-4405-aaf8-45e5cac0110c.png">

<img width="1449" alt="Screen Shot 2022-11-21 at 10 46 58 AM" src="https://user-images.githubusercontent.com/3297292/203098934-82fc2364-9b8f-4c2b-9ca5-722b5280da67.png">

Reply:
Pushed a fix for this, however other things aren't yet working. Suggest we merge and continue working on this.